packages/react/src/ReactDebugCurrentFrame.js

```diff
@@ -7,26 +7,20 @@
  * @flow
  */
```

Comment:
The organization of this file is confusing to me.
1. We unconditionally export a setter which sets a variable that is not exported (and has no getter or is otherwise unused *except* for DEV mode which uses it in `getStackAddendum`).
2. We assign a duplicate of that same setter to `ReactDebugCurrentFrame` in DEV mode, but we don't assign a getter. We leave that up to the current renderer, which of course, doesn't even use the local value written by `ReactDebugCurrentFrame.setExtraStackFrame` because it has no way to access it. So instead it uses its own local current fiber value.

Looks like the only thing using the exported `setCurrentlyValidatingElement` function (now `setExtraStackFrame`) is `ReactElementValidator` - and **even it only uses it in DEV**. So why are we using this odd pattern?

I'm not super familiar with some of these files so maybe there's a subtle difference that I'm overlooking.

Suggestion: Update `ReactElementValidator` to use `ReactDebugCurrentFrame.*` too, then get rid of the ES export.

Reply:
It's complicated because this is used by `jsx-runtime` too which is a separate file that needs to import this from isomorphic. This needs some further refactoring in a follow up at some point.

For now I'm just removing a cycle that otherwise would break.